contrib/netperf/collectd-td.spec.in

```diff
@@ -1833,8 +1836,16 @@ if [ $1 -eq 2 ]; then
 fi
 %systemd_post collectd.service
 %else
-/sbin/chkconfig --add cgconfig || :
-/sbin/chkconfig --add collectd-td || :
+/sbin/chkconfig --add cgconfig ||
+    echo 2>&1 '"chkconfig --add cgconfig" failed (keep going)'
+/sbin/chkconfig --add collectd-td ||
+    echo 2>&1 '"chkconfig --add collectd" failed (keep going)'
+levels=`/bin/sed -r -n 's/^ *# *chkconfig: *([0-9]+).*$/\1/p' \
```

Comment:
Just to be sure, the new commands run the service on installation? Prior to that it wasn't run following installation?